GitUI/Editor/FileViewer.Designer.cs

```diff
@@ -99,7 +99,7 @@ private void InitializeComponent()
             this.cherrypickSelectedLinesToolStripMenuItem.Image = global::GitUI.Properties.Images.CherryPick;
             this.cherrypickSelectedLinesToolStripMenuItem.Name = "cherrypickSelectedLinesToolStripMenuItem";
             this.cherrypickSelectedLinesToolStripMenuItem.Size = new System.Drawing.Size(243, 22);
-            this.cherrypickSelectedLinesToolStripMenuItem.Text = "Cherry pick selected lines";
+            this.cherrypickSelectedLinesToolStripMenuItem.Text = Strings.StageSelectedLines;
```

Comment:
If you move this out, then it won't get translated second time.

Reply:
Move out how?
I looked a little at the TranslationApp, did not find an obvious way to fix this.